packages/eslint-plugin/src/rules/ban-context-export/index.test.js

```diff
@@ -3,43 +3,40 @@ const { RuleTester } = require('@typescript-eslint/rule-tester');
 const path = require('path');
 const rule = require('./index');
 
-const ruleTester = new RuleTester({
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    project: path.resolve(__dirname, './fixtures/ban-context-export/tsconfig.json'),
-    tsconfigRootDir: path.resolve(__dirname, './fixtures/ban-context-export'),
-  },
-});
+const ruleTester = new RuleTester();
 
 /**
  * @param {string} fixtureName
  */
 function getParserOptions(fixtureName) {
   return {
-    project: path.resolve(__dirname, `./fixtures/${fixtureName}/tsconfig.json`),
-    tsconfigRootDir: path.resolve(__dirname, `./fixtures/${fixtureName}`),
+    parserOptions: {
+      parser: '@typescript-eslint/parser',
+      project: path.resolve(__dirname, `./fixtures/${fixtureName}/tsconfig.json`),
+      tsconfigRootDir: path.resolve(__dirname, `./fixtures/${fixtureName}`),
+    },
   };
 }
 
 ruleTester.run('ban-context-export', rule, {
   valid: [
     {
-      parserOptions: getParserOptions('internal-export'),
+      languageOptions: getParserOptions('internal-export'),
```

Comment:
according to release notes: https://typescript-eslint.io/blog/announcing-typescript-eslint-v8/#flat-configuration-ruletester